src/main/java/net/shadowmage/ancientwarfare/npc/registry/FactionNpcDefault.java

```diff
@@ -90,8 +92,13 @@ public FactionNpcDefault setThinness(float thinness) {
 		return change(def -> def.thinness = thinness);
 	}
 
+	public FactionNpcDefault setSpells(String spells) {
```

Comment:
I am not a big fan of having this functionality here given that it's a pretty core class to NPCs and the functionality is for mod compatibility, but I don't have any immediate ideas of how to take care of it. Could you please leave a TODO comment here basically saying this should likely move to a better place